scottypy/app.py

```diff
@@ -146,16 +151,14 @@ def down(beam_id_or_tag, dest, url, overwrite, filter):  # pylint: disable=W0622
 
     if beam_id_or_tag.startswith('t:'):
         tag = beam_id_or_tag[2:]
-        if dest is None:
-            dest = tag
+        dest = dest or tag
 
         for beam in scotty.get_beams_by_tag(tag):
-            _download_beam(beam, os.path.join(dest, str(beam.id)), overwrite, filter)
+            asyncio.get_event_loop().run_until_complete(_download_beam(beam, os.path.join(dest, str(beam.id)), overwrite, filter))
```

Comment:
These can be done concurrently, you should call `run_until_complete()` once instead of in a loop.